### PR TITLE
Updated index.md in release-3.0  to direct users to the latest 4.0 documentation

### PR DESCRIPTION
**Pull request by loadinglit**
fixes #1960 

- Documentation Update: This pull request updates the index.md file for Release 3.0 to direct users to the latest documentation, ensuring they have access to the most current and relevant information.

- SEO Consideration: In light of the recent SEO issues affecting search engine indexing, this update is crucial to prevent users from landing on outdated documentation pages

Note: In a separate pull request for the Release 4.0 branch, I am proposing an update to the robots.txt file. This change is designed to prevent search engines from indexing older versions of the documentation, thereby enhancing the visibility and accessibility of the most current documentation for users.

### CODE REVIEW

**Comment by jrobinAV** [comment]
Hi, I believe your latest PR covers the issue. This one should be closed, right?

**Review by jrobinAV — CHANGES_REQUESTED** [review]
.gitignore should not be committed

**Comment by loadinglit** [comment]
> .gitignore should not be committed

@jrobinAV pls check, its undone now, let me know if any change

File: docs/knowledge_base/tips/using_tables/index.md
Comment (jrobinAV) [inline docs/knowledge_base/tips/using_tables/index.md:1]: Why this file?
Why only this file?

**Review by jrobinAV — CHANGES_REQUESTED** [review]

File: docs/knowledge_base/tips/using_tables/index.md
Comment (loadinglit) [reply docs/knowledge_base/tips/using_tables/index.md:1]: Pls check this pr - https://github.com/Avaiga/taipy/issues/1960#issue-2572873346
Caching issues result in showing up the release version 3.0 of taipy tables 
- Incase this happens it notifies the user to refer to the latest doc available

Comment (jrobinAV) [reply docs/knowledge_base/tips/using_tables/index.md:1]: Oh, ok I understand why you pick this file, then. 

However, I believe @AlexandreSajus talked about this page as an example. What we want is to have such a message on all the pages. Ideally, that could be added automatically to all the documentation pages as a sticky banner.

What do you think?

Comment (loadinglit) [reply docs/knowledge_base/tips/using_tables/index.md:1]: Thanks for the clarification! That makes sense. Do you have an example of a sticky banner that would serve as a reference? I'm particularly interested in how to customize it for each documentation page.

Comment (loadinglit) [reply docs/knowledge_base/tips/using_tables/index.md:1]: Also pls assign me this issue , i'd like to work on it

**Pull request closed without merging**